examples/sample-deployments/kong-enterprise-hybrid/cluster-tls/cluster.key

```diff
@@ -0,0 +1,6 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
I suppose it is an example of a private key, but... can we make it even clear?

Like:
```
 -----BEGIN CERTIFICATE----- 
EXAMPLE
 -----END CERTIFICATE----- 
```

Reply:
Ok we can do that